Netable/Example/ExampleApp.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ExampleApp.swift
+//  Example
+//
+//  Created by Amy Oulton on 2022-11-23.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import SwiftUI
+
+@main
+struct ExampleApp: App {
+    init() {
+        ExampleNetworkService.shared
```

Comment:
This network calling can be improved upon, it's just a simple way to get it going now.